cds/config.py

```diff
@@ -317,6 +317,10 @@ def _(x):
 # Endpoint for record ui.
 RECORDS_UI_ENDPOINT = '{scheme}://{host}/record/{pid_value}'
 
+# IIIF
+#: Improve quality of image resampling using better algorithm
+IIIF_RESIZE_RESAMPLE = 'PIL.Image:BICUBIC'
```

Comment:
Is this variable used anywhere? I can't find any reference to it in the `invenio-iiif` repo (https://github.com/inveniosoftware/invenio-iiif/search?q=IIIF_RESIZE_RESAMPLE&unscoped_q=IIIF_RESIZE_RESAMPLE).

Reply:
It is in `flask-iiif`: https://github.com/inveniosoftware/flask-iiif/blob/master/flask_iiif/config.py#L62

It is the default value, we could actually remove it...